lib/tasks/populate_topics.rake

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'avro/builder'
```

Comment:
Why this?

Reply:
@jpslav must have been a copy paste error from somewhere. removed.